example_site/src/components/VideoInfo.js

```diff
@@ -0,0 +1,36 @@
+import "./VideoInfo.css"
+
+/**
+ * Renders a video component showing video thumbnail and information
+ * 
+ * @param {Object} video Video object
+ */
+const VideoInfo = ({ video }) => {
+    var createdDate = (video.createdAt + "").substr(0, 15);
```

Comment:
Use const not var

Reply:
Let's talk about using `.substr()` here once we figure out exactly what the Date objects look like